app/search/query/base.py

```diff
@@ -89,7 +87,7 @@ def get_result_abstract(self, result):
         """
         return self._strip_html(result['abstract'])
 
-    @memorize(prefix='query', error_on_miss=True)
+    @memorize(prefix='query')
```

Comment:
This seems like a significant change; I'm not sure how errors will be thrown if a search fails. Should memorize.py set `error_on_miss=True` by default? Or is there a corresponding change I'm missing in the flow of the main search method?